### PR TITLE
Add emoji selector

### PR DESCRIPTION
**Pull request by jrainville**
Ads a basic emoji selector.

Pending points:
- Right now , the way I made it work is a did a quick script in a Node REPL, got the file names of all the emojis and created an array of those file name. With that array, I can add the emojis to a Model in QML.
  - This means that we don't have categories yet, though it would be easy to support with by making an object instead of an array. Something like this:
```json
{
  "category1": ["icon1", "..."],
  "categorty2": ["..."]
}
```
 - However, doing this manually sounds like the biggest pain in the world
   - **Question**: Do you guys know a way we could do it automatically?
    - Consider that the file names are actually the codepoints of the emojis. Meaning that the names are not random, they are the Hex codepoint that can be converted to the UTF bytes that show the icon.

- Another point is that right now, we remove the sticker and emoji buttons when we start entering text
  - So does that means that we only want to send single emojis with no text? IMO, that's a bad idea.
  - I think it would be better to leave the emoji button next to the send button because emojis can be added with text (that's how the emoji selector works aright now anyway, it appends the emoji to the text area)
  - I'll ask @simonricoo about it

image

### CODE REVIEW

**Comment by jrainville** [comment]
@iurimatias Fixed.
TLDR, flag icons and some others have a `-` in their name, and the Emoji parser doesn't handle that, it only encodes the first part. So I need to split myself and encode.

I also fixed a problem where the emojis didn't ellide in the last message thing in the contact column
